assessments/KRAssessment/templates/studentAssessments.jsont

```diff
@@ -0,0 +1,132 @@
+ {% if OverallScore is not none and OverallScore | length %}
```

Comment:
@Bruk real quick - have you tested that this logic actually works and validates? Like does the file you're running against have an example where overall score is missing? It might be better to just filter those records out in the earthmover.yaml file, I think this logic would leave an empty row which may cause issues during validation. But I may be wrong!